### PR TITLE
DOCS-3048: Refine Server to server documentation

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/orders/self-hosted-credit-card.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+> When no 3D verification is required, the [transaction status](/about-payments/multisafepay-statuses/) response will be processed directly and no form will be sent.
+
+```json 
+{
+  "success":true,
+  "data":{
+    ...
+  }
+}
+```
```
Comment (Sarah-MultiSafepay) [inline content/api/orders/self-hosted-credit-card.md:141]: Should this be here?

File: content/api/orders/self-hosted-credit-card.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+---
+{{< code-block >}}
+> POST - /orders
+
+```json
+{
+  "type":"direct",
+  "gateway":"MASTERCARD",
+  "order_id":"order-1",
```
Comment (Sarah-MultiSafepay) [inline content/api/orders/self-hosted-credit-card.md:13]:
```suggestion
  "order_id":"my-order-id-1",
```

File: content/api/orders/self-hosted-credit-card.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+      "first_name":"Testperson-nl",
+      "house_number":"XI",
+      "last_name":"Approved",
+      "locale":"nl",
+      "phone1":"0612345678",
+      "phone2":"",
+      "state":null,
+      "zip_code":"2521VA"
+    },
+    "description":"product description",
```
Comment (Sarah-MultiSafepay) [inline content/api/orders/self-hosted-credit-card.md:84]:
```suggestion
    "description":"Test order description",
```

File: content/api/orders/self-hosted-credit-card.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+    "fastcheckout":"NO",
+    "financial_status":"initialized",
+    "items":null,
+    "modified":"2022-01-31T14:37:26",
+    "order_id":"apitool_4767937",
```
Comment (Sarah-MultiSafepay) [inline content/api/orders/self-hosted-credit-card.md:89]:
```suggestion
    "order_id":"my-order-id-1",
```

File: content/api/orders/self-hosted-credit-card.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+    "payment_details":{
```
Comment (Sarah-MultiSafepay) [inline content/api/orders/self-hosted-credit-card.md:90]: The payment details don't match what was sent in the request.

File: content/api/orders/self-hosted-credit-card.md [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+  }
+}
+```
+{{< /code-block >}}
+
+{{< description >}}
+
+## Self-hosted credit card payments 
+
+See [Self-hosted credit card payments](self-hosted-credit-card-payments).
```
Comment (Sarah-MultiSafepay) [inline content/api/orders/self-hosted-credit-card.md:148]:
```suggestion
See [Self-hosted credit card payments](/features/self-hosted-credit-card-payments).
```

File: content/payments/features/server-to-server/_index.md [hunk 7]
```diff
@@ -1,15 +1,57 @@
...
+    - /features/server-to-server/integration/
+    - /features/server-to-server/
 ---
 
-Server to Server is a MultiSafepay solution that lets customers make credit card payments in your checkout page without being redirected to a MultiSafepay payment page.
+The usual way of processing credit card payments is to redirect customers to a [MultiSafepay payment page](/payment-pages/). In this case, MultiSafepay is responsible for handling customers' sensitive credit card payment details. Our server is fully [PCI DSS compliant](/payment-regulations/pci-dss/). 
+
+If you want to avoid redirecting customers to a MultiSafepay payment page to reduce friction and increase conversion, you can:
+
+- Use a [payment component](/payment-components/), where MultiSafepay remains responsible for PCI DSS compliance.
```
Comment (Sarah-MultiSafepay) [inline content/payments/features/server-to-server/_index.md:27]: Should we capitalise "payment component" like we do on the Payment Components page?

**Review by Sarah-MultiSafepay — COMMENTED** [review]
This is looking much better @FrancescaNCook!

File: content/payments/features/server-to-server/_index.md (hunk 7 above)
Comment (FrancescaNCook) [inline content/payments/features/server-to-server/_index.md:27]:
```suggestion
- Use a [Payment Component](/payment-components/), where MultiSafepay remains responsible for PCI DSS compliance.

```

File: content/api/orders/self-hosted-credit-card.md (hunk 5 above)
Comment (FrancescaNCook) [reply content/api/orders/self-hosted-credit-card.md]: Fixed! Thanks!

File: content/api/orders/self-hosted-credit-card.md (hunk 1 above)
Comment (FrancescaNCook) [reply content/api/orders/self-hosted-credit-card.md]: No! Removed. Thanks!

File: content/api/orders/flexible-3d.md [hunk 8]
```diff
@@ -129,25 +127,27 @@ meta_title: "API reference - Direct: Flexible 3D enabled - MultiSafepay Docs"
...
 
-Use [Flexible 3D](/payments/features/flexible-3ds/) to set whether or not to complete the transaction with [3D Secure](/security-and-legal/payment-regulations/about-3d-secure/) verification.
-
-To enable 3D Secure, in the `POST /orders` request > `gateway_info` object, set the `flexible_3d` parameter to `true`.
+{{< alert-notice >}} We no longer support Flexible 3D for merchants based in Europe due to PSD2 regulations. {{< /alert-notice >}}
 
-**Notes:**  
-
-- Activating Flexible 3D Secure overrides Dynamic 3D settings, so that payments are not enrolled with a 3D authentication.
+Use [Flexible 3D](/payments/features/flexible-3ds/) to set whether or not to complete the transaction with [3D Secure](/security-and-legal/payment-regulations/about-3d-secure/) verification.
```
[the remark this answers is not in the document]
Comment (FrancescaNCook) [reply content/api/orders/flexible-3d.md:134]: Consider in another ticket.

**Review by Kris-MultiSafepay — DISMISSED** [review]
Looks very good!

File: content/api/orders/credit-card-direct.md [hunk 9]
```diff
@@ -0,0 +1,346 @@
...
+  }
+}
+```
+{{< /code-block >}}
+
+{{< description >}}
+
+### Credit card – direct 
+
+See [Handling cardholder data](/about-payments/handling-cardholder-data/).
```
Comment (Kris-MultiSafepay) [inline content/api/orders/credit-card-direct.md:135]:
```suggestion
**Note**: To make credit card – direct requests, you need to process cardholder data on your server, which requires PCI DSS certification. We recommend using a [MultiSafepay payment page](/payment-pages/) or [Payment Component](/payment-components/) to accept credit card payments. 

See also, [Handling cardholder data](/about-payments/handling-cardholder-data/). 
```

**Review by Kris-MultiSafepay — APPROVED** [review]

**Review by Sarah-MultiSafepay — APPROVED** [review]

**Pull request merged**